src/guides/v2.2/performance-best-practices/software.md

```diff
@@ -95,6 +95,18 @@ opcache.memory_consumption=64
 opcache.max_accelerated_files=60000
 ```
 
+#### APCU
+
+We recommend enabling the [PHP APCu extension](https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-2-b-apcu-cache) and [configuring `composer` to support it]({{ page.baseurl }}/performance-best-practices/deployment-flow.html#preprocess-dependency-injection-instructions) to optimize for maximum performance. This extension caches file locations for opened files, which increases performance for Magento server calls including pages, ajax calls, and endpoints.
+
+Edit your `apcu.ini` file to include the following:
+
+```text
+extension=apcu.so
+[apcu]
+acp.enabled = 1
+```
+
```

Comment:
Does it matter where these lines are placed?

Reply:
In the default .ini file, the acp.enabled is already there at the top, so it's just adding the extension.